tailwind.config.ts

```diff
@@ -53,6 +53,13 @@ export default {
             backgroundImage: {
                 lino: 'url("$lib/lino-hero.svg")',
             },
+            height: {
+                '136': '34rem',
+            },
+            width: {
+                '104': '26rem',
+                '156': '39rem',
+            },
```

Comment:
These seem like one-off arbitrary values rather than something we should encode into the Tailwind configuration. I think the inline `[...]` style syntax is more appropriate here.